tardis/montecarlo/src/cmontecarlo.c

```diff
@@ -194,46 +194,46 @@ compute_distance2line (const rpacket_t * packet, const storage_model_t * storage
-	    {
-	      fprintf (stderr, "last_line = %f\n",
-		       storage->
-		       line_list_nu[rpacket_get_next_line_id (packet) - 1]);
-	      fprintf (stderr, "Last line in line list reached!");
-	    }
-	  else if (rpacket_get_next_line_id (packet) == 0)
-	    {
-	      fprintf (stderr, "First line in line list!");
-	      fprintf (stderr, "next_line = %f\n",
-		       storage->
-		       line_list_nu[rpacket_get_next_line_id (packet) + 1]);
-	    }
-	  else
-	    {
-	      fprintf (stderr, "last_line = %f\n",
-		       storage->
-		       line_list_nu[rpacket_get_next_line_id (packet) - 1]);
-	      fprintf (stderr, "next_line = %f\n",
-		       storage->
-		       line_list_nu[rpacket_get_next_line_id (packet) + 1]);
-	    }
-	  fprintf (stderr, "ERROR: Comoving nu less than nu_line!\n");
-	  fprintf (stderr, "comov_nu = %f\n", comov_nu);
-	  fprintf (stderr, "nu_line = %f\n", nu_line);
-	  fprintf (stderr, "(comov_nu - nu_line) / nu_line = %f\n",
-		   (comov_nu - nu_line) / nu_line);
-	  fprintf (stderr, "r = %f\n", r);
-	  fprintf (stderr, "mu = %f\n", mu);
-	  fprintf (stderr, "nu = %f\n", nu);
-	  fprintf (stderr, "doppler_factor = %f\n", doppler_factor);
-	  fprintf (stderr, "cur_zone_id = %" PRIi64 "\n", cur_zone_id);
-	  ret_val = TARDIS_ERROR_COMOV_NU_LESS_THAN_NU_LINE;
-	}
+        {
+          if (rpacket_get_next_line_id (packet) == storage->no_of_lines - 1)
+            {
+              fprintf (stderr, "last_line = %f\n",
+                       storage->
+                       line_list_nu[rpacket_get_next_line_id (packet) - 1]);
+              fprintf (stderr, "Last line in line list reached!");
+            }
+          else if (rpacket_get_next_line_id (packet) == 0)
+            {
+              fprintf (stderr, "First line in line list!");
+              fprintf (stderr, "next_line = %f\n",
+                       storage->
+                       line_list_nu[rpacket_get_next_line_id (packet) + 1]);
+            }
+          else
+            {
+              fprintf (stderr, "last_line = %f\n",
+                       storage->
+                       line_list_nu[rpacket_get_next_line_id (packet) - 1]);
+              fprintf (stderr, "next_line = %f\n",
+                       storage->
+                       line_list_nu[rpacket_get_next_line_id (packet) + 1]);
+            }
+          fprintf (stderr, "ERROR: Comoving nu less than nu_line!\n");
+          fprintf (stderr, "comov_nu = %f\n", comov_nu);
+          fprintf (stderr, "nu_line = %f\n", nu_line);
+          fprintf (stderr, "(comov_nu - nu_line) / nu_line = %f\n",
+                   (comov_nu - nu_line) / nu_line);
+          fprintf (stderr, "r = %f\n", r);
+          fprintf (stderr, "mu = %f\n", mu);
+          fprintf (stderr, "nu = %f\n", nu);
+          fprintf (stderr, "doppler_factor = %f\n", doppler_factor);
+          fprintf (stderr, "cur_zone_id = %" PRIi64 "\n", cur_zone_id);
```

Comment:
That is for printing the correct type for `int64_t` PRIi64 get's expanded to the apropiate character combination, most likely `l` so in the end it's `%l`